src/DMXSerial2.h

```diff
@@ -155,16 +159,17 @@ struct RDMSENSOR {
   int16_t normalMax;
   bool8 lowHighSupported;
   bool8 recordedSupported;
-  char *description;
+  const char *description;
 }; // struct RDMSENSOR
 
 struct RDMINIT {
   const char          *manufacturerLabel; //
   const uint16_t          deviceModelId;       //
   const char          *deviceModel;       //
-  uint16_t footprint;
-  // uint16_t personalityCount;
-  // RDMPERSONALITY *personalities;
+//  uint16_t footprint; // This now depends on the personality data
+  uint8_t defaultPersonalityNumber; // Is this excessive flexibility?
```

Comment:
I've had personal feedback suggesting it's not and probably matches commercial fixtures, so I'm tempted to leave this...